src/interactive/repl.ts

```diff
@@ -180,6 +180,10 @@ const g_onInit = new vscode.EventEmitter<rpc.MessageConnection>()
 export const onInit = g_onInit.event
 const g_onExit = new vscode.EventEmitter<Boolean>()
 export const onExit = g_onExit.event
+const g_onStartEval = new vscode.EventEmitter<null>()
+export const onStartEval = g_onStartEval.event
+const g_onFinishEval = new vscode.EventEmitter<null>()
+export const onFinishEval = g_onFinishEval.event
```

Comment:
`rpc.onMessageConnection.onNotification` only allow one single hook to be registered, so we need to setup our own event hooks for them.